### DISCUSSION TITLE
Customization of show indicator

### DISCUSSION THREAD

**Discussion by DottorBooom** [domferr/tilingshell#124] [Ideas]
Hello, 
as you can read, is it possible to add a voice in the appearance settings when you can change the color of the indicator or the opacity, the blurriness in a scale of 0% to 100%? Cause it would be nice if the user could also work on what the indicator looks like.
I have another one, in editing layouts when you want to divide windows and the user wants a perfect half partition you need to scroll until you get the exact number of pixels in both indicators. Is it possible to add a shortcut for a perfect split both vertically and horizontally?

**Comment by domferr** [comment]
Hi, thank you for sharing these ideas!

> is it possible to add a voice in the appearance settings when you can change the color of the indicator or the opacity, the blurriness in a scale of 0% to 100%?

I'm not sure this is something I may work on, since there are many other tiling features to still implement. Will see!
Meanwhile you can achieve the same by editing the indicator file, which is in the extension's files.

> when you want to divide windows and the user wants a perfect half partition you need to scroll until you get the exact number of pixels in both indicators. Is it possible to add a shortcut for a perfect split both vertically and horizontally?

This is something I'll work on, follow the issue #103 for updates on this!
